app/commands/detail/route.js

```diff
@@ -11,6 +11,15 @@ export default Route.extend({
     ]).then(arr => {
       const [verPayload, tagPayload] = arr;
 
+      if (params.version) {
+        const exsistVersion = verPayload.filter(t => t.version === params.version);
```

Comment:
```suggestion
        const existVersion = verPayload.filter(t => t.version === params.version);
```
or `existingVersion`

Reply:
Thanks! I fixed!